components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -72,6 +88,37 @@ export default function Preferences() {
                         </p>
                     </div>
                 </div>
+
+                <h3 className="mt-12">Inactivity Timeout </h3>
+                <p className="text-base text-gray-500 dark:text-gray-400">
+                    By default, workspaces stop following 30 minutes without user input (e.g. keystrokes or terminal
+                    input commands). You can increase the workspace timeout up to a maximum of 24 hours.
+                </p>
+                <div className="mt-4 max-w-xl">
+                    <h4>Default Workspace Inactivity Timeout</h4>
+                    <span className="flex">
+                        <input
+                            type="text"
+                            className="w-96 h-9"
+                            value={workspaceTimeout}
+                            disabled={!allowConfigureWorkspaceTimeout}
+                            placeholder="timeout time, such as 30m, 1h, max 24h"
```

Comment:
**Suggestion(non blocking):** @gtsiolis - The placeholder text here quite verbose, and duplicate with the helper text below. Could we simplify?

|Before|After|
|-|-|
|<img width="518" alt="image" src="https://user-images.githubusercontent.com/5528307/222167803-2ce15cd5-b6df-46b7-adf1-8fe40d9bf827.png">|<img width="519" alt="image" src="https://user-images.githubusercontent.com/5528307/222167682-b26ac886-875b-4e99-b344-da58f3bfde9e.png">|

Reply:
Cross-posting some ideas on how we could simplify this in https://github.com/gitpod-io/gitpod/pull/16503#discussion_r1122082069, but this can be certainly out of the scope here.

> Long-term, I'd like to see this moving away from this custom value and rely on just a number, making this a number type of field. It's quite confusing and hard to understand what's going on here, whereas if we just used minutes as we do with prebuilds it could simplify things, including what copy we use for labels and input help text.

This could look like the following: 

| BEFORE | AFTER |
|-|-|
| <img width="815" alt="timeout-before" src="https://user-images.githubusercontent.com/120486/222216112-35d56841-2663-429e-864f-6329bbbe4fb4.png"> | <img width="815" alt="timeout-after" src="https://user-images.githubusercontent.com/120486/222216115-8808357f-a25d-4bee-bf9a-98834bc7df4b.png"> |

Thanks for the ping, @loujaybee! 🏓

Reply:
Okay, happy to also fix in follow up 🙏